### PR TITLE
feat(cli) kong prepare cli command to prepare prefix

### PR DESCRIPTION
**Pull request by thefosk**
### Full changelog

* Introduces a new `kong prepare` command that only prepares the prefix.

Documentation at https://github.com/Mashape/getkong.org/commit/c5b332ef0e0484b36ecf807fd812bc66ff7e234a

### CODE REVIEW

File: kong/cmd/prepare.lua [hunk 1]
```diff
@@ -0,0 +1,30 @@
+local prefix_handler = require "kong.cmd.utils.prefix_handler"
+local conf_loader = require "kong.conf_loader"
+local log = require "kong.cmd.utils.log"
+
+local function execute(args)
+  local conf = assert(conf_loader(args.conf, {
+    prefix = args.prefix
+  }))
+
+  local ok, err = prefix_handler.prepare_prefix(conf, args.nginx_conf)
```
Comment (thibaultcha) [inline kong/cmd/prepare.lua]: I think we are missing this `nginx-conf` argument from the command's options?

File: kong/cmd/prepare.lua [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  if not ok then
+    log.verbose("could not prepare Kong")
+    error(err) -- report to main error handler
+  end
+end
+
+local lapp = [[
+Usage: kong prepare [OPTIONS]
+
+Prepares the Kong prefix in the configured prefix directory.
```
Comment (thibaultcha) [inline kong/cmd/prepare.lua]: The CLI's help messages do not use the third person. Should be `Prepare the Kong prefix...`

File: kong/cmd/prepare.lua [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+local function execute(args)
+  local conf = assert(conf_loader(args.conf, {
+    prefix = args.prefix
+  }))
+
+  local ok, err = prefix_handler.prepare_prefix(conf, args.nginx_conf)
+  if not ok then
+    log.verbose("could not prepare Kong")
+    error(err) -- report to main error handler
```
Comment (thibaultcha) [inline kong/cmd/prepare.lua]: A better experience would be to log:
```lua
if not ok then
  error("could not prepare Kong prefix at " .. conf.prefix .. 
        ": " .. err)
end
```

File: kong/cmd/prepare.lua (hunk 2 above)
Comment (thibaultcha) [inline kong/cmd/prepare.lua]: The help message should also include an Example usage like the `kong compile` command does, to help the user understand the use-case that this command solves:
```
Prepare the Kong prefix in the configured prefix directory. This command can
be used to start Kong from the nginx binary without using the 'kong start' command.

Example usage:
  kong prepare -p /usr/local/kong -c kong.conf && nginx -p /usr/local/kong -c nginx.conf
```

File: kong/cmd/prepare.lua (hunk 1 above)
Comment (thefosk) [inline kong/cmd/prepare.lua]: Addressed in last commit.

**Comment by thefosk** [comment]
@Tieske we decided to keep both for the time being

**Comment by Tieske** [comment]
@thefosk ok, but then a deprecation warning should be added to the docs

**Comment by thefosk** [comment]
@Tieske Deprecated with https://github.com/Mashape/getkong.org/pull/432/commits/a68553c17611757986224c628c6cf94adeeafdb5

**Pull request merged**